src/analysis/locally_convex/with_seminorms.lean

```diff
@@ -453,13 +453,14 @@ section continuous_bounded
 
 namespace seminorm
 
-variables [normed_field 𝕜] [add_comm_group E] [module 𝕜 E] [add_comm_group F] [module 𝕜 F]
+variables [nontrivially_normed_field 𝕜] [normed_field 𝕝] [add_comm_group E] [module 𝕜 E]
+variables [module 𝕝 E] [add_comm_group F] [module 𝕜 F] [module 𝕝 F]
```

Comment:
Sorry for the stupid question, but is there a case where you need both `𝕜` and `𝕝`?